includes/functions.php

```
@@ -647,3 +647,127 @@ function lsx_to_sanitize_tour_title( $title = '' ) {
+
+	/**
+	 * Filters the SVG icon content.
+	 *
+	 * @since 2.1.0
+	 *
+	 * @param string $svg_content The SVG content.
+	 * @param string $icon_type   The icon type.
+	 * @param string $icon_name   The icon name.
+	 */
+	return apply_filters( 'lsx_to_icon_svg', $svg_content, $icon_type, $icon_name );
+}
```

PHPMD (2.15.0)

The function lsx_to_get_icon_svg() has 108 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)